README.md

```diff
@@ -34,43 +64,70 @@ clouds:
       user_domain_name: ldap_users
     identity_api_version: 3
     cacert: /usr/share/pki/trust/anchors/SUSE_Trust_Root.crt.pem
+ansible:
+  use_hostnames: True
+  expand_hostvars: False
+  fail_on_errors: True
+```
+
+If you don't have the SUSE root certificate installed, check
+http://ca.suse.de/.
 
 You'll also need to pre-create some configuration in engcloud. It's
 convention here to use your username as part of the name of objects you create.
 
-Using the engcloud web interface or OpenStack CLI (i.e. "openstack
-keypair create..."), create a keypair
+Create a keypair on engcloud (using either the engcloud web interface or
+OpenStack CLI's `openstack keypair create`)
 for accessing the instances created. Remember the name of this
-keypair.
+keypair (named further `foctodoodle-key`)
```

Comment:
I'm guessing "named further" is roughly equivalent to "which appears as 'foctodoodle-key' in the example below". Which is, of course, more verbose, but clearer.

Reply:
Thanks! Will fix!